tasks/main.yml

```diff
@@ -60,7 +60,7 @@
 
     - set_fact:
         he_shutdown_cmd: >-
-          while hosted-engine --vm-status | grep "\"vm\": \"up\"" >/dev/null; do sleep 5; done; shutdown -h now
+          while hosted-engine --vm-status | grep "\"vm\": \"up\"" >/dev/null; do sleep 5; done; sanlock client shutdown -f 1; shutdown -h now
```

Comment:
OK for me, but as discussed in private, I think we should also (separate patch) handle errors from '--vm-status'. IIUC, if it writes to stderr, after the engine vm is down, and with it ssh is dead, and so is its stdout/stderr, the bash command running on the host will get a sigpipe and will die.

Reply:
I tend to say that pseudo-terminal allocation with ssh -t should be enough to avoid that.
What can instead fail is that we have a simple positive check on "vm: up" and with everything else (including errors) we trigger an immediate shutdown.
Probably it will be safer to invert the logic and trigger the shutdown only when we are really sure that the engine VM is really down.